tests/test_authentication.py

```diff
@@ -140,13 +142,19 @@ def test_unknown_creds(self):
                               '^access denied: CredentialsLookupError: ')
 
     def test_alternative_credential_lookup(self):
-        sender = self._sender(credentials=ALTERNATIVE_CREDS)
-        req = self._request(sender)
         lookup_path = "%s.alternative_lookup" % __name__
         with self.settings(HAWK_CREDENTIALS_LOOKUP=lookup_path):
+            # Reload with settings change, re-instantiate
+            reload_module(hawkrest)
```

Comment:
Yeah, that was me modifying the test _before_ adding in the subclassable methods. Will go back and redo that. (I get an icky feeling about `reload_module()` too.)